webknossos-datastore/app/com/scalableminds/webknossos/datastore/controllers/MeshController.scala

```diff
@@ -66,12 +77,81 @@ class MeshController @Inject()(
                 "mesh.file.listChunks.failed",
                 request.body.segmentId.toString,
                 request.body.meshFile) ?~> Messages("mesh.file.load.failed", request.body.segmentId.toString) ~> BAD_REQUEST
```

Comment:
Is this still necessary?

Reply:
Well spotted, this was dead code, I removed it now